pyproject.toml

```diff
@@ -62,6 +62,9 @@ black = "^21.7b0"
 isort = "^5.9.3"
 doc8 = "0.9.0"
 
+[tool.poetry.extras]
+toml = ["tomli"]
+
```

Comment:
Wait, whats the difference between what you do here and what I setup in line 48 where I specify this as optional...

oh.. is this what allows `pip install diff_cover[tomli]` ?  In that case I would just say `toml` here. In the off chance we change how the toml stuff works down the line I would not want to have to change this.

Reply:
Exactly, you got it. `tomli = {version = "^1.2.1", optional = true}` is just the pure dependency. Then `toml = ["tomli"]` declares `tomli` has an extra dependency which can be installed with `pip install diff_cover[toml]` (not tomli)